packages/apps-config/package.json

```diff
@@ -14,6 +14,7 @@
     "@edgeware/node-types": "^3.3.4",
     "@equilab/definitions": "1.0.3",
     "@interlay/polkabtc-types": "^0.6.2",
+    "@kiltprotocol/type-definitions": "latest",
```

Comment:
This is great. Could you rather point to the version - the reason being is that I do update the dependencies (weekly, sometimes more) and `yarn upgrade-interactive` I can see what got bumped. It is unlikely that a `rm -rf yarn.lock` is called.

Reply:
Sure! No problem. I was wondering which would be preferred. 👍  I have updated it now.